Gimp failed to build on macOS High Sierra

## Issue description
fyi @jtojnar get:
```
- Error: missing dependency glib-networking
      *** Test for glib-networking failed. This is required.
```

Note also that I have 2.8 installed with homebrew - possible issue?

```
~$ nix-env -i -A nixpkgs.gimp
warning: name collision in input Nix expressions, skipping '/Users/bencreasy/.nix-defexpr/channels_root/nixpkgs'
warning: name collision in input Nix expressions, skipping '/Users/bencreasy/.nix-defexpr/channels_root/nixpkgs-unstable'
warning: name collision in input Nix expressions, skipping '/Users/bencreasy/.nix-defexpr/channels_root/root-unstable'
installing 'gimp-2.10.2'
these derivations will be built:
  /nix/store/x53bsvixx2jai1ccdqnj8iy7dk5rr8qi-gimp-2.10.2.drv
these paths will be fetched (37.65 MiB download, 75.53 MiB unpacked):
  /nix/store/0bdnvr90lr1na92k0hn603kzdfwqsjsv-babl-0.1.50
  /nix/store/1g7xl2zwlyarq2m5493kbsfxkhdssx0m-gtk+-2.24.32-dev
  /nix/store/22v14c7g6lls1hxfd1dd92yj3qwklp62-gexiv2-0.10.8
  /nix/store/3fx370zda2zczkv8kxs4z3799j0j92na-gtk+-2.24.32
  /nix/store/3vdfk1lqyfwv7i8alxrqch94zgiwjkxz-atk-2.28.1-dev
  /nix/store/bas3rr331mi8ad6k8hsshz4jjks68c96-libwmf-0.2.8.4
  /nix/store/gam66jz1nldihmmpwvy305wvjw3kg14d-gegl-0.4.2
  /nix/store/gv4qghjvx07y1xqwncy8bwffqz4fh3m9-librsvg-2.42.2-dev
  /nix/store/gx4f08bhr9qhm0yxqdmfhh7swpd6nb5m-gtk-mac-integration-2.0.8
  /nix/store/if9zxfkh1wq6h47jmlkn0ki6a4vkscm9-librsvg-2.42.2
  /nix/store/j5vlkkv64p7ppq27j4xvh7qn5aihlc5z-atk-2.28.1
  /nix/store/q9h8r3ys16a5xn5y41a6kczz4116kjx6-python2.7-pygtk-2.24.0
  /nix/store/wf1n426xmpzx79jggapfnmbc2my9f6wr-gimp-2.10.2.tar.bz2
copying path '/nix/store/wf1n426xmpzx79jggapfnmbc2my9f6wr-gimp-2.10.2.tar.bz2' from 'https://cache.nixos.org'...
copying path '/nix/store/0bdnvr90lr1na92k0hn603kzdfwqsjsv-babl-0.1.50' from 'https://cache.nixos.org'...
copying path '/nix/store/22v14c7g6lls1hxfd1dd92yj3qwklp62-gexiv2-0.10.8' from 'https://cache.nixos.org'...
copying path '/nix/store/if9zxfkh1wq6h47jmlkn0ki6a4vkscm9-librsvg-2.42.2' from 'https://cache.nixos.org'...
copying path '/nix/store/bas3rr331mi8ad6k8hsshz4jjks68c96-libwmf-0.2.8.4' from 'https://cache.nixos.org'...
copying path '/nix/store/j5vlkkv64p7ppq27j4xvh7qn5aihlc5z-atk-2.28.1' from 'https://cache.nixos.org'...
copying path '/nix/store/3vdfk1lqyfwv7i8alxrqch94zgiwjkxz-atk-2.28.1-dev' from 'https://cache.nixos.org'...
copying path '/nix/store/3fx370zda2zczkv8kxs4z3799j0j92na-gtk+-2.24.32' from 'https://cache.nixos.org'...
copying path '/nix/store/gam66jz1nldihmmpwvy305wvjw3kg14d-gegl-0.4.2' from 'https://cache.nixos.org'...
copying path '/nix/store/gv4qghjvx07y1xqwncy8bwffqz4fh3m9-librsvg-2.42.2-dev' from 'https://cache.nixos.org'...
copying path '/nix/store/1g7xl2zwlyarq2m5493kbsfxkhdssx0m-gtk+-2.24.32-dev' from 'https://cache.nixos.org'...
copying path '/nix/store/gx4f08bhr9qhm0yxqdmfhh7swpd6nb5m-gtk-mac-integration-2.0.8' from 'https://cache.nixos.org'...
copying path '/nix/store/q9h8r3ys16a5xn5y41a6kczz4116kjx6-python2.7-pygtk-2.24.0' from 'https://cache.nixos.org'...
building '/nix/store/x53bsvixx2jai1ccdqnj8iy7dk5rr8qi-gimp-2.10.2.drv'...
unpacking sources
unpacking source archive /nix/store/wf1n426xmpzx79jggapfnmbc2my9f6wr-gimp-2.10.2.tar.bz2
source root is gimp-2.10.2
setting SOURCE_DATE_EPOCH to timestamp 1526841866 of file gimp-2.10.2/devel-docs/libgimp/libgimp.types
patching sources
autoreconfPhase
autoreconf: Entering directory `.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force -I m4macros ${ACLOCAL_FLAGS}
autoreconf: configure.ac: tracing
autoreconf: running: libtoolize --copy --force
libtoolize: putting auxiliary files in '.'.
libtoolize: copying file './ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'm4macros'.
libtoolize: copying file 'm4macros/libtool.m4'
libtoolize: copying file 'm4macros/ltoptions.m4'
libtoolize: copying file 'm4macros/ltsugar.m4'
libtoolize: copying file 'm4macros/ltversion.m4'
libtoolize: copying file 'm4macros/lt~obsolete.m4'
autoreconf: running: /nix/store/x7v4bw0y8zsja14f64v3vjdny8inlb7y-autoconf-2.69/bin/autoconf --force
autoreconf: running: /nix/store/x7v4bw0y8zsja14f64v3vjdny8inlb7y-autoconf-2.69/bin/autoheader --force
autoreconf: running: automake --add-missing --copy --force-missing
configure.ac:237: installing './compile'
configure.ac:98: installing './missing'
app-tools/Makefile.am: installing './depcomp'
autoreconf: Leaving directory `.'
configuring
fixing libtool script ./ltmain.sh
configure flags: --disable-static --disable-dependency-tracking --prefix=/nix/store/1wcd1lpxrgpl1wfxlz09wkc5ia71xa43-gimp-2.10.2 --without-webkit --with-bug-report-url=https://github.com/NixOS/nixpkgs/issues/new --with-icc-directory=/var/run/current-system/sw/share/color/icc
checking for a BSD-compatible install... /nix/store/5lkrw9dnsgy62qm1ampvww1c5n1pdm4b-coreutils-8.29/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /nix/store/5lkrw9dnsgy62qm1ampvww1c5n1pdm4b-coreutils-8.29/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether make supports nested variables... (cached) yes
checking for gcc... clang
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether clang accepts -g... yes
checking for clang option to accept ISO C89... none needed
checking whether clang understands -c and -o together... yes
checking whether make supports the include directive... yes (GNU style)
checking dependency style of clang... none
checking how to run the C preprocessor... clang -E
checking build system type... x86_64-apple-darwin17.5.0
checking host system type... x86_64-apple-darwin17.5.0
checking for gcc... no
checking for cc... cc
checking whether we are using the GNU C compiler... (cached) yes
checking whether cc accepts -g... yes
checking for cc option to accept ISO C89... (cached) none needed
checking whether cc understands -c and -o together... (cached) yes
checking dependency style of cc... (cached) none
checking how to run the C preprocessor... cc -E
checking whether we are using the GNU C++ compiler... yes
checking whether clang++ accepts -g... yes
checking dependency style of clang++... none
checking whether clang++ supports C++14 features by default... no
checking whether clang++ supports C++14 features with -std=gnu++14... yes
checking how to print strings... printf
checking for a sed that does not truncate output... /nix/store/llk6iyb3ppvyaflbkg5rbv960wlsxpkc-gnused-4.4/bin/sed
checking for grep that handles long lines and -e... /nix/store/c3sa761qjcaa5rg0s8h8qxj0mghkidv5-gnugrep-3.1/bin/grep
checking for egrep... /nix/store/c3sa761qjcaa5rg0s8h8qxj0mghkidv5-gnugrep-3.1/bin/grep -E
checking for fgrep... /nix/store/c3sa761qjcaa5rg0s8h8qxj0mghkidv5-gnugrep-3.1/bin/grep -F
checking for ld used by clang... ld
checking if the linker (ld) is GNU ld... no
checking for BSD- or MS-compatible name lister (nm)... nm
checking the name lister (nm) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 196608
checking how to convert x86_64-apple-darwin17.5.0 file names to x86_64-apple-darwin17.5.0 format... func_convert_file_noop
checking how to convert x86_64-apple-darwin17.5.0 file names to toolchain format... func_convert_file_noop
checking for ld option to reload object files... -r
checking for objdump... no
checking how to recognize dependent libraries... pass_all
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for archiver @FILE support... no
checking for strip... strip
checking for ranlib... ranlib
checking command to parse nm output from clang object... ok
checking for sysroot... no
checking for a working dd... /nix/store/5lkrw9dnsgy62qm1ampvww1c5n1pdm4b-coreutils-8.29/bin/dd
checking how to truncate binary pipes... /nix/store/5lkrw9dnsgy62qm1ampvww1c5n1pdm4b-coreutils-8.29/bin/dd bs=4096 count=1
checking for mt... no
checking if : is a manifest tool... no
checking for dsymutil... dsymutil
checking for nmedit... no
checking for lipo... lipo
checking for otool... otool
checking for otool64... no
checking for -single_module linker flag... yes
checking for -exported_symbols_list linker flag... yes
checking for -force_load linker flag... yes
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if clang supports -fno-rtti -fno-exceptions... yes
checking for clang option to produce PIC... -fno-common -DPIC
checking if clang PIC flag -fno-common -DPIC works... yes
checking if clang static flag -static works... no
checking if clang supports -c -o file.o... yes
checking if clang supports -c -o file.o... (cached) yes
checking whether the clang linker (ld) supports shared libraries... yes
checking dynamic linker characteristics... darwin17.5.0 dyld
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... no
checking how to run the C++ preprocessor... clang++ -std=gnu++14 -E
checking for ld used by clang++ -std=gnu++14... ld
checking if the linker (ld) is GNU ld... no
checking whether the clang++ -std=gnu++14 linker (ld) supports shared libraries... yes
checking for clang++ -std=gnu++14 option to produce PIC... -fno-common -DPIC
checking if clang++ -std=gnu++14 PIC flag -fno-common -DPIC works... yes
checking if clang++ -std=gnu++14 static flag -static works... no
checking if clang++ -std=gnu++14 supports -c -o file.o... yes
checking if clang++ -std=gnu++14 supports -c -o file.o... (cached) yes
checking whether the clang++ -std=gnu++14 linker (ld) supports shared libraries... yes
checking dynamic linker characteristics... darwin17.5.0 dyld
checking how to hardcode library paths into programs... immediate
checking dependency style of clang... none
checking whether to enable maintainer-specific portions of Makefiles... yes
checking for pkg-config... /nix/store/3jg66rxk2zrs7whk6d6pifkq26cy7y0k-pkg-config-0.29.2/bin/pkg-config
checking pkg-config is at least version 0.16... yes
checking for host type... x86_64-apple-darwin17.5.0
checking for host processor... x86-64
checking if compiling for Win32... no
checking if compiling for Mac OS X... yes
yes
checking whether make sets $(MAKE)... (cached) yes
checking whether ln -s works... yes
checking for special C compiler options needed for large files... no
checking for _FILE_OFFSET_BITS value needed for large files... no
checking whether to turn on debugging... no
checking whether to turn on profiling... no
checking whether clang understands -Wdeclaration-after-statement... yes
checking whether clang understands -Wmissing-prototypes... yes
checking whether clang understands -Werror=missing-prototypes... yes
checking whether clang understands -Wstrict-prototypes... yes
checking whether clang understands -Wmissing-declarations... yes
checking whether clang understands -Winit-self... yes
checking whether clang understands -Wpointer-arith... yes
checking whether clang understands -Wold-style-definition... yes
checking whether clang understands -Wmissing-format-attribute... yes
checking whether clang understands -Wformat-security... yes
checking whether clang understands -Wlogical-op... no
checking whether clang understands -Wtype-limits... yes
checking whether clang understands -fno-common... yes
checking whether clang understands -fdiagnostics-show-option... yes
checking whether clang understands -Wreturn-type... yes
checking whether clang understands --omg-optimized... no
checking whether clang understands --disable-instructions... no
checking for ANSI C header files... (cached) yes
checking for sys/wait.h that is POSIX.1 compatible... yes
checking whether time.h and sys/time.h may both be included... yes
checking execinfo.h usability... yes
checking execinfo.h presence... yes
checking for execinfo.h... yes
checking fcntl.h usability... yes
checking fcntl.h presence... yes
checking for fcntl.h... yes
checking sys/param.h usability... yes
checking sys/param.h presence... yes
checking for sys/param.h... yes
checking sys/prctl.h usability... no
checking sys/prctl.h presence... no
checking for sys/prctl.h... no
checking sys/thr.h usability... no
checking sys/thr.h presence... no
checking for sys/thr.h... no
checking sys/time.h usability... yes
checking sys/time.h presence... yes
checking for sys/time.h... yes
checking sys/times.h usability... yes
checking sys/times.h presence... yes
checking for sys/times.h... yes
checking for sys/wait.h... (cached) yes
checking for unistd.h... (cached) yes
checking for backtrace... yes
checking for pid_t... yes
checking for vprintf... yes
checking for _doprnt... no
checking for size_t... yes
checking for working alloca.h... yes
checking for alloca... yes
checking for fsync... yes
checking for difftime... yes
checking for mmap... yes
checking for thr_self... no
checking whether binary relocation support should be enabled... no
checking for _NL_MEASUREMENT_MEASUREMENT... no
checking whether NLS is requested... yes
checking for intltool >= 0.40.1... 0.51.0 found
checking for intltool-update... /nix/store/v110fximniyx1i1hq4i0a7xcj5i8fjb8-intltool-0.51.0/bin/intltool-update
checking for intltool-merge... /nix/store/v110fximniyx1i1hq4i0a7xcj5i8fjb8-intltool-0.51.0/bin/intltool-merge
checking for intltool-extract... /nix/store/v110fximniyx1i1hq4i0a7xcj5i8fjb8-intltool-0.51.0/bin/intltool-extract
checking for xgettext... /nix/store/s8yvb556vw7l73qzx2wmhc2qmdw972dy-gettext-0.19.8/bin/xgettext
checking for msgmerge... /nix/store/s8yvb556vw7l73qzx2wmhc2qmdw972dy-gettext-0.19.8/bin/msgmerge
checking for msgfmt... /nix/store/s8yvb556vw7l73qzx2wmhc2qmdw972dy-gettext-0.19.8/bin/msgfmt
checking for gmsgfmt... /nix/store/s8yvb556vw7l73qzx2wmhc2qmdw972dy-gettext-0.19.8/bin/msgfmt
checking for perl... /nix/store/lizm7q36j5sf5524bw9gsykflv9i2l7z-perl-5.24.3/bin/perl
checking for perl >= 5.8.1... 5.24.3
checking for XML::Parser... ok
checking locale.h usability... yes
checking locale.h presence... yes
checking for locale.h... yes
checking for LC_MESSAGES... yes
checking for CFPreferencesCopyAppValue... yes
checking for CFLocaleCopyCurrent... yes
checking libintl.h usability... yes
checking libintl.h presence... yes
checking for libintl.h... yes
checking for ngettext in libc... yes
checking for dgettext in libc... yes
checking for bind_textdomain_codeset... yes
checking for msgfmt... (cached) /nix/store/s8yvb556vw7l73qzx2wmhc2qmdw972dy-gettext-0.19.8/bin/msgfmt
checking for dcgettext... yes
checking if msgfmt accepts -c... yes
checking for gmsgfmt... (cached) /nix/store/s8yvb556vw7l73qzx2wmhc2qmdw972dy-gettext-0.19.8/bin/msgfmt
checking for xgettext... (cached) /nix/store/s8yvb556vw7l73qzx2wmhc2qmdw972dy-gettext-0.19.8/bin/xgettext
checking for recommended xgettext version (>= 0.19)... yes
checking for iso-codes... yes
checking for babl >= 0.1.50... yes
checking for gegl-0.4 >= 0.4.2... yes
checking for atk >= 2.2.0... yes
checking for pkg-config... (cached) /nix/store/3jg66rxk2zrs7whk6d6pifkq26cy7y0k-pkg-config-0.29.2/bin/pkg-config
checking pkg-config is at least version 0.16... yes
checking for GLIB - version >= 2.54.2... yes (version 2.56.0)
checking for gmodule-no-export-2.0... yes
checking for gio-2.0... yes
checking for gio-unix-2.0... yes
checking for glib-networking (GIO TLS implementation)... no
configure: Eeeeeeeeeeeeeeeeeeeeek! Missing dep: glib-networking
checking for glib-compile-resources... /nix/store/dkmxfrzsi0bzg4a4b976ww08243fslbs-glib-2.56.0-dev/bin/glib-compile-resources
checking for gdbus-codegen... /nix/store/dkmxfrzsi0bzg4a4b976ww08243fslbs-glib-2.56.0-dev/bin/gdbus-codegen
checking if GLib is version 2.43.0 or newer... yes
checking for gegl... /nix/store/gam66jz1nldihmmpwvy305wvjw3kg14d-gegl-0.4.2/bin/gegl
checking for bind_textdomain_codeset... (cached) yes
checking for X... libraries , headers
checking for gethostbyname... yes
checking for connect... yes
checking for remove... yes
checking for shmat... yes
checking for IceConnectionNumber in -lICE... yes
checking for pkg-config... (cached) /nix/store/3jg66rxk2zrs7whk6d6pifkq26cy7y0k-pkg-config-0.29.2/bin/pkg-config
checking pkg-config is at least version 0.7... yes
checking for GTK+ - version >= 2.24.10... yes (version 2.24.32)
checking for gtk-update-icon-cache... /nix/store/3fx370zda2zczkv8kxs4z3799j0j92na-gtk+-2.24.32/bin/gtk-update-icon-cache
checking if GTK+ is version 2.26.0 or newer (bogus placeholder check)... no
checking for gdk-pixbuf-2.0 >= 2.30.8... yes
checking for gdk-pixbuf-csource... /nix/store/v1grr57j36kf9g8pmiihlvbwdp03s2hj-gdk-pixbuf-2.36.7-dev/bin/gdk-pixbuf-csource
checking for cairo >= 1.12.2... yes
checking for pangocairo >= 1.29.4 pangoft2... yes
checking for fontconfig >= 2.12.4... yes
checking if Pango is version 1.32.0 or newer... yes
checking if Pango is built with a recent fontconfig... yes
checking for freetype2 >= 2.1.7... yes
checking for harfbuzz >= 0.9.19... yes
checking for gexiv2 >= 0.10.6... yes
checking for execinfo.h... (cached) yes
checking math.h usability... yes
checking math.h presence... yes
checking for math.h... yes
checking ieeefp.h usability... no
checking ieeefp.h presence... no
checking for ieeefp.h... no
checking for extra flags to get ANSI library prototypes... none needed
checking for finite... yes
checking for rint... yes
checking for vfork... yes
checking for __attribute__((destructor))... yes
checking for getaddrinfo... yes
checking for getnameinfo... yes
checking for socket in -lsocket... no
checking whether clang understands -mfpmath=sse... yes
checking whether clang understands -msse2... yes
checking whether we can compile SSE2 intrinsics... yes
checking whether clang understands -msse4.1... yes
checking whether we can compile SSE4.1 intrinsics... yes
checking whether clang understands -mmmx... yes
checking whether we can compile MMX code... yes
checking whether clang understands -msse... yes
checking whether we can compile SSE code... yes
checking for shm_open... yes
checking sys/mman.h usability... yes
checking sys/mman.h presence... yes
checking for sys/mman.h... yes
checking for shared memory transport type... posix
checking whether symbols are prefixed... yes, with a _
checking fd_set and sys/select... yes
checking for TIFFReadScanline in -ltiff... yes
checking tiffio.h usability... yes
checking tiffio.h presence... yes
checking for tiffio.h... yes
checking for jpeg_destroy_decompress in -ljpeg... yes
checking for jpeglib.h... yes
checking for jpeg_save_markers in -ljpeg... yes
checking for gzsetparams in -lz... yes
checking zlib.h usability... yes
checking zlib.h presence... yes
checking for zlib.h... yes
checking bzlib.h usability... yes
checking bzlib.h presence... yes
checking for bzlib.h... yes
checking for BZ2_bzCompress in -lbz2... yes
checking for liblzma >= 5.0.0... yes
checking for gsapi_new_instance in -lgs... yes
checking ghostscript/iapi.h usability... yes
checking ghostscript/iapi.h presence... yes
checking for ghostscript/iapi.h... yes
checking for libpng >= 1.6.25... yes
checking for mng_create in -lmng... yes
checking libmng.h usability... yes
checking libmng.h presence... yes
checking for libmng.h... yes
checking for aa_printf in -laa... yes
checking aalib.h usability... yes
checking aalib.h presence... yes
checking for aalib.h... yes
checking for XpmReadFileToXpmImage in -lXpm... yes
checking X11/xpm.h usability... yes
checking X11/xpm.h presence... yes
checking for X11/xpm.h... yes
checking for OpenEXR >= 1.6.1... yes
checking for libwebp >= 0.6.0... yes
checking for libwebpmux >= 0.6.0... yes
checking for libwebpdemux >= 0.6.0... yes
checking for libheif >= 1.1.0... no
checking for libmypaint >= 1.3.0... yes
checking for mypaint-brushes-1.0... yes
checking whether to treat MyPaint brushes as bundled... no
checking for librsvg-2.0 >= 2.40.6... yes
checking for poppler-glib >= 0.44.0... yes
checking for poppler-data >= 0.4.7... yes
checking for cairo-pdf >= 1.12.2... yes
checking for libwmf-config... /nix/store/bas3rr331mi8ad6k8hsshz4jjks68c96-libwmf-0.2.8.4/bin/libwmf-config
checking for libwmf >= 0.2.8... yes (version 0.2.8)
checking for libopenjp2 >= 2.1.0... yes
checking for lcms2 >= 2.8... yes
checking for xcursor... yes
checking for ALSA CFLAGS...
checking for ALSA LDFLAGS...  -lasound -lm -ldl -lpthread
checking for libasound headers version >= 1.0.0... not present.
checking for snd_ctl_open in -lasound... no
checking linux/input.h usability... no
checking linux/input.h presence... no
checking for linux/input.h... no
checking for xdg-email... no
checking for perl5... no
checking for perl... /nix/store/lizm7q36j5sf5524bw9gsykflv9i2l7z-perl-5.24.3/bin/perl
checking for perl version... 5.24.3
checking for a Python 2 interpreter with version >= 2.5.0... python
checking for python... /nix/store/10f4qxb00wk9j6cicd9h1qzwqgnc36ji-python-2.7.14/bin/python
checking for python version... 2.7
checking for python platform... darwin
checking for python script directory... ${prefix}/lib/python2.7/site-packages
checking for python extension module directory... ${exec_prefix}/lib/python2.7/site-packages
checking for headers required to compile python extensions... found
checking for pygtk-2.0 >= 2.10.4... yes
checking for pygtk-codegen-2.0... /nix/store/q9h8r3ys16a5xn5y41a6kczz4116kjx6-python2.7-pygtk-2.24.0/bin/pygtk-codegen-2.0
checking for pygtk defs... /nix/store/q9h8r3ys16a5xn5y41a6kczz4116kjx6-python2.7-pygtk-2.24.0/share/pygtk/2.0/defs
checking for pycairo >= 1.0.2... yes
checking whether clang understands -fno-strict-aliasing... yes
checking for gtk-mac-integration-gtk2 >= 2.0.0... yes
checking for xmllint... no
checking for xsltproc... no
checking for shared-mime-info... yes
checking for xvfb-run... no
checking for appstream-util... no
checking for desktop-file-validate... no
checking for gtk-doc... no
configure: WARNING:
  You will not be able to create source packages with 'make dist'
  because gtk-doc >= 1.0 is not found.
checking for gtkdoc-check... no
checking for gtkdoc-check... no
checking for gtkdoc-rebase... no
checking for gtkdoc-mkpdf... no
checking whether to build gtk-doc documentation... no
checking for glib-2.0 >= 2.10.0 gobject-2.0  >= 2.10.0... yes
checking if building the Windows installer... no
configure: Generating POTFILES and Makefile for po-libgimp
configure: Generating POTFILES and Makefile for po-python
configure: Generating POTFILES and Makefile for po-plug-ins
configure: Generating POTFILES and Makefile for po-script-fu
configure: Generating POTFILES and Makefile for po-tips
configure: Generating POTFILES and Makefile for po-windows-installer
configure: error:

Extra Binaries:
  gimp-console:            yes

Optional Features:
  Language selection:      yes
  Vector icons:            yes
  Dr. Mingw (Win32):       no
  Bundled MyPaint Brushes: no (at /nix/store/c1vshikx6v2lidfds3i95839zjzc7m8p-mypaint-brushes-1.3.0/share/mypaint-data/1.0/brushes)
  Default ICC directory:   /var/run/current-system/sw/share/color/icc

Optional Plug-Ins:
  Ascii Art:               yes
  Ghostscript:             yes
  Help Browser:            no (WebKit support disabled)
  JPEG 2000:               yes
  MNG:                     yes
  OpenEXR:                 yes
  WebP:                    yes
  Heif:                    no (libheif not found)
  PDF (export):            yes
  Print:                   yes
  Python 2:                yes
  TWAIN (Win32):           no
  Webpage:                 no (WebKit support disabled)
  WMF:                     yes
  X11 Mouse Cursor:        yes
  XPM:                     yes
  Email:                   needs runtime dependency: xdg-email

Optional Modules:
  ALSA (MIDI Input):       no (libasound not found or unusable)
  Linux Input:             no (linux input support disabled) (GUdev support: no)
  DirectInput (Win32):     no

Tests:
  Use xvfb-run             no (not found)
  Test appdata             no (appstream-util not found)
  Test desktop file        no (desktop-file-validate not found)

Bug report URL: https://github.com/NixOS/nixpkgs/issues/new


Error: GIMP configuration failed.

  - Error: missing dependency glib-networking
      *** Test for glib-networking failed. This is required.

See the file 'INSTALL' for more help.
builder for '/nix/store/x53bsvixx2jai1ccdqnj8iy7dk5rr8qi-gimp-2.10.2.drv' failed with exit code 1
error: build of '/nix/store/x53bsvixx2jai1ccdqnj8iy7dk5rr8qi-gimp-2.10.2.drv' failed
```

### Steps to reproduce

nix-env -i -A nixpkgs.gimp

## Technical details
running High Sierra (17E199)
````
 - system: `"x86_64-darwin"`
 - host os: `Darwin 17.5.0, macOS 10.13.4`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.0.2`
 - channels(bencreasy): `"nixpkgs, nixpkgs-unstable"`
 - channels(root): `"nixpkgs, nixpkgs-unstable, root-unstable"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixpkgs`
````

`nix show-config` (not sure if that's helpful)

````
allow-import-from-derivation = true
allow-unsafe-native-code-during-evaluation = false
allowed-impure-host-deps = /System/Library /bin/sh /dev /usr/lib
allowed-uris =
allowed-users = *
auto-optimise-store = false
build-hook = /nix/store/hs8mxsvdhm95dxgx943d74fws01j2zj3-nix-2.0.2/libexec/nix/build-remote
build-poll-interval = 5
build-users-group = nixbld
builders = @/etc/nix/machines
builders-use-substitutes = false
compress-build-log = true
connect-timeout = 0
cores = 1
darwin-log-sandbox-violations = false
diff-hook =
enforce-determinism = true
extra-sandbox-paths =
extra-substituters =
fallback = false
fsync-metadata = true
gc-check-reachability = false
gc-reserved-space = 8388608
hashed-mirrors = http://tarballs.nixos.org/
http-connections = 25
http2 = true
impersonate-linux-26 = false
keep-build-log = true
keep-derivations = true
keep-env-derivations = false
keep-failed = false
keep-going = false
keep-outputs = false
max-build-log-size = 0
max-free = 18446744073709551615
max-jobs = 32
max-silent-time = 0
min-free = 0
netrc-file = /etc/nix/netrc
plugin-files =
pre-build-hook = /nix/store/hs8mxsvdhm95dxgx943d74fws01j2zj3-nix-2.0.2/libexec/nix/resolve-system-dependencies
print-missing = true
pure-eval = false
repeat = 0
require-sigs = true
restrict-eval = false
run-diff-hook = false
sandbox = false
sandbox-paths =
secret-key-files =
show-trace = false
store = daemon
substitute = true
substituters = https://cache.nixos.org/
sync-before-registering = false
system = x86_64-darwin
tarball-ttl = 3600
timeout = 0
trusted-public-keys = cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
trusted-substituters =
trusted-users = root
use-case-hack = true
use-sqlite-wal = true
user-agent-suffix =
````

--- a/pkgs/applications/graphics/gimp/default.nix
+++ b/pkgs/applications/graphics/gimp/default.nix
@@ -3,7 +3,7 @@
 , libmng, librsvg, libwmf, zlib, libzip, ghostscript, aalib, shared-mime-info
 , python2Packages, libexif, gettext, xorg, glib-networking, libmypaint, gexiv2
 , harfbuzz, mypaint-brushes, libwebp, libheif, libgudev, openexr
-, AppKit, Cocoa, gtk-mac-integration }:
+, AppKit, Cocoa, gtk-mac-integration-gtk2, cf-private }:
 
 let
   inherit (python2Packages) pygtk wrapPython python;
@@ -23,8 +23,11 @@ in stdenv.mkDerivation rec {
     freetype fontconfig lcms libpng libjpeg poppler poppler_data libtiff openexr
     libmng librsvg libwmf zlib libzip ghostscript aalib shared-mime-info libwebp libheif
     python pygtk libexif xorg.libXpm glib-networking libmypaint mypaint-brushes
-  ] ++ stdenv.lib.optionals stdenv.isDarwin [ AppKit Cocoa gtk-mac-integration ]
-    ++ stdenv.lib.optionals stdenv.isLinux [ libgudev ];
+  ] ++ stdenv.lib.optionals stdenv.isDarwin [
+    # cf-private is needed to get some things not in swift-corefoundation.
+    # For instance _OBJC_CLASS_$_NSArray is missing.
+    AppKit Cocoa gtk-mac-integration-gtk2 cf-private
+  ] ++ stdenv.lib.optionals stdenv.isLinux [ libgudev ];
 
   pythonPath = [ pygtk ];
 
@@ -69,7 +72,9 @@ in stdenv.mkDerivation rec {
     "--with-icc-directory=/var/run/current-system/sw/share/color/icc"
   ];
 
-  doCheck = true;
+  # on Darwin,
+  # test-eevl.c:64:36: error: initializer element is not a compile-time constant
+  doCheck = !stdenv.isDarwin;
 
   enableParallelBuilding = true;
 

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -17050,6 +17050,7 @@ with pkgs;
     lcms = lcms2;
     inherit (gnome3) gexiv2;
     inherit (darwin.apple_sdk.frameworks) AppKit Cocoa;
+    inherit (darwin) cf-private;
   };
 
   gimp-with-plugins = callPackage ../applications/graphics/gimp/wrapper.nix {

